snake_game_L1.py

```diff
@@ -105,12 +105,24 @@ def move_right():
 
     while True:
         simulakra.update()
-        if head.xcor() > 750 or head.xcor() < -750 or head.ycor() > 375 or head.ycor() < -375:
-            px = head.xcor()
-            nx = -px
-            py = head.ycor()
-            ny = -py
-            head.goto(nx, ny)
+        if head.xcor() > 750:
+            head.goto(-750, head.ycor())
+
+        if head.xcor() < -750:

```

Comment:
- eğer `head` nesnesinin x koordinatı -750'yi geçerse neler olacağını belirlemek için bir `if` bloğu başlatıyoruz